docs/pages/preview/eas-update/deployment-patterns.md

```diff
@@ -0,0 +1,152 @@
+- Previous versions of your deployments are always preserved on GitHub. Once the `version-1.0` branch is deployed, then another version is deployed after it (like `version-1.1`), the `version-1.0` branch is forever preserved, making it easy to checkout a previous version of your project.
+
+Disadvantages of this flow:
+
+- Bookkeeping of branch names is required for this flow, which will mean communicating with your team which branches are currently pointed at your test builds and your production builds.
+
+## Persistent staging flow
+
+This flow is a slight variation of the "Branch flow". The main difference is that it does not require bookkeeping branches. Instead, we'll have a persistent `staging` and `production` branches we can merge into forever. It's great for medium- projects to large-sized projects. Here are the parts of the deployment process above that make up this flow:
+
+Creating builds: (b) Create builds for production use and separate builds for testing use.
+
+Testing changes: (a) Test changes on TestFlight and the Play Store Internal Track and/or (b) Test changes with internal distribution builds.
+
+Publishing updates: (c) create update branches that are environment based, like `staging` and `production`.
+
+Diagram of flow:
+
+![Staging deployment diagram](/static/images/eas-update/deployment-staging.png)
+
+Explanation of flow:
+
+- Develop a project locally and test changes in Expo Go.
+- Create builds with channels named `production` , which will eventually get reviewed and become available on app stores. Create another set of builds with channels named `staging`, which will be used for testing on TestFlight and the Play Store Internal Track.
+- Set up expo-github-action to publish updates when merging commits to branches.
+- Merge changes into a branch named `staging`. The GitHub Action will publish an update and make it available on our test builds.
+- When ready, merge changes into the `production` branch to publish an update to our production builds.
+
+Advantages of this flow:
+
+- This flow is an even balance between safety and speed.
+- It's easy to communicate to your team, since deploying updates occurs when merging into GitHub branches named `staging` and `production`.
+
+Disadvantages of this flow:
+
+- Checking out previous versions of your app is slightly more complex, since we'd need to check out an old commit instead of an old branch.
+- When merging to `production`, the update would be re-built and re-published instead of moved from the builds with channel `staging` to the builds with channel `production`.
+
+## Platform specific flow
+
+This flow is for projects that need to build and update their iOS and Android apps separately all the time. It will result in separate commands for delivering updates to the iOS and Android apps. Here are the parts of the deployment process above that make up this flow:
```

Comment:
```suggestion
This flow is for projects that need to build and update their Android and iOS apps separately all the time. It will result in separate commands for delivering updates to the Android and iOS apps. Here are the parts of the deployment process above that make up this flow:
```